src/pages/Dashboard/Families/FamilyDetail/FamilyInfo.tsx

```diff
@@ -0,0 +1,36 @@
+import type { MantineColor } from '@mantine/core';
+import { Badge, Box, Space, Text, Title } from '@mantine/core';
+
+interface Props {
+  title: string;
+  value: string;
+  isBadge?: boolean;
+  badgeStatus?: Extract<MantineColor, 'green' | 'red' | 'yellow'>;
```

Comment:
I think the `isBadge` is extra 
but even if we want to keep it (which I don't encourage), the current typing allows having `isBadge=false`  with `badgeStatus='red'`

Reply:
It isn't extra. We must use it when we have a value that it is a badge based on the design.